sentinelsat/sentinel.py

```diff
@@ -1149,15 +1149,15 @@ def geojson_to_wkt(geojson_obj, feature_number=0, decimals=4):
         a GeoJSON object
     feature_number : int, optional
         Feature to extract polygon from (in case of MultiPolygon
-        FeatureCollection), defaults to first Feature
+        FeatureCollection), defaults to first feature
     decimals : int, optional
         Number of decimal figures after point to round coordinate to. Defaults to 4 (about 10
         meters).
 
     Returns
     -------
-    polygon coordinates
-        string of comma separated coordinate tuples (lon, lat) to be used by SentinelAPI
+    str
+        Well-Known Text string corresponding to the GeoJSON geometry.
```

Comment:
Niggle: A WKT is a geometry representation that in itself is not related to GeoJSON, except that they both refer to spatial geometries. The phrasing sounds like this is somehow a WKT+GeoJSON string.

Reply:
Good point. Is "Well-Known Text string representation of the geometry" better?

Reply:
Sounds good :+1: